docs/pages/management/operations/upgrading.mdx

```diff
@@ -53,13 +53,22 @@ Back up before upgrading. We have more instructions in [Backing up Teleport](./b
 <TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
 When upgrading a single Teleport cluster:
 
-1. **Upgrade the Auth Service first**. The Auth Service keeps the cluster state
+<Notice type="tip">
```

Comment:
I think we can incorporate this information into the ordered list below the `Notice` so it's explicit to users what they need to do when. Otherwise, a user needs to fill in some blanks for themselves in order to connect the `Notice` to the ordered list. 

Also, the `Notice` seems to cover what the ordered list implies in step 4, but it's not completely explicit that this is the case.

I think we should create a single ordered list that includes the information in the `Notice`.